doc/api/esm.md

```diff
@@ -602,20 +609,22 @@ CommonJS modules loaded.
 * `specifier` {string}
 * `context` {Object}
   * `conditions` {string[]}
-  * `parentURL` {string}
-* `defaultResolve` {Function}
+  * `parentURL` {string|undefined}
+* `defaultResolve` {Function} The Node.js default resolver.
 * Returns: {Object}
-  * `url` {string}
+  * `format` {string|undefined} `'builtin'|'commonjs'|'json'|'module'|'wasm'`
```

Comment:
`null` is a valid value in this PR implementation: https://github.com/nodejs/node/pull/37468/files#diff-f6796082f599554ec3a29c47cf026cb24fc5104884f2632e472c05fe622d778bR436-R439
```suggestion
  * `format` {string|undefined|null} `'builtin'|'commonjs'|'json'|'module'|'wasm'`
```

Reply:
Oh 🤦 Sorry, I was thinking this was `load`'s `context.format` (which would be either string or undefined). Yes indeed. Thank you.

p.s ugh, those extra characters make the line now exceed 80 characters.

Reply:
Oh, actually, I see that value is just passed along (so it could be null in load's context.format). Would it be worthwhile to restricted to either string or undefined? 🤔